modules/wrapping/dev.galasa.wrapping.io.grpc.java/pom.xml

```diff
@@ -19,71 +19,71 @@
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-api</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-context</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-core</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-netty-shaded</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 		</dependency>
 		<dependency>
 			<groupId>io.grpc</groupId>
 			<artifactId>grpc-protobuf</artifactId>
-			<version>1.68.0</version>
+			<version>1.76.0</version>
 
 			<!-- 
 			This dependency has a transient dependency of this:
-			com.google.protobuf:protobuf-java:jar:3.25.3:compile
+			com.google.protobuf:protobuf-javalite:jar:3.25.8:compile
```

Comment:
Minor - this comment has been changed to `protobuf-javalite`, should it be `protobuf-java`?